process/elasticproc/logsevents/logsAndEventsProcessor.go

```diff
@@ -47,32 +46,31 @@ func NewLogsAndEventsProcessor(args *ArgsLogsAndEventsProcessor) (*logsAndEvents
 }
 
 func checkArgsLogsAndEventsProcessor(args *ArgsLogsAndEventsProcessor) error {
-	if check.IfNil(args.ShardCoordinator) {
-		return elasticIndexer.ErrNilShardCoordinator
-	}
 	if check.IfNil(args.PubKeyConverter) {
-		return elasticIndexer.ErrNilPubkeyConverter
+		return dataindexer.ErrNilPubkeyConverter
```

Comment:
send the args as value, or check `args` for `nil`

Reply:
changed to value